src/adaXT/decision_tree/splitter.pyx

```diff
@@ -24,7 +24,7 @@ cdef int compare(const void* a, const void* b) noexcept nogil:
     else:
         return -1
 
-cdef int[::1] sort_feature(int[::1] indices):
+cdef inline int[::1] sort_feature(int[::1] indices):
```

Comment:
What does this do?

Reply:
It is an optimization for the smaller functions, where instead of calling the function, it is able to take the inner code from the function and use it in place: https://www.ibm.com/support/pages/what-does-it-mean-inline-function-and-how-does-it-affect-program